theme/themes/eea/views/card.overrides

```diff
@@ -249,6 +252,62 @@
   }
 }
 
+/*******************************
+      Image with title
+*******************************/
+// Adding the default image css for image inside image-with-title
+.ui.cards > .card > .image, .ui.card .image-with-title > .image {
+  position: relative;
+  display: block;
+  flex: 0 0 auto;
+  padding: 0em;
+  background: transparent;
+}
+
+.image-with-title {
+  position: relative;
+}
+
+// Gradient Mixin
+.gradientMix(@type; @colors; @dir: 0deg; @prefixes: webkit, moz, ms, o; @index: length(@prefixes)) when (@index > 0) {
+  .gradientMix(@type; @colors; @dir; @prefixes; (@index - 1));
+
+  @prefix  : extract(@prefixes, @index);
+  @dir-old : 90 - (@dir);
+
+  background-image: ~"-@{prefix}-@{type}-gradient(@{dir-old}, @{colors})";
+& when ( @index = length(@prefixes) ) {
+      background-image: ~"@{type}-gradient(@{dir}, @{colors})";
+  }
+}
+
+.image-with-title .gradient {
+  display: flex;
+  align-items: end;
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  height: 100%;
+  padding: @imageTitlePadding;
+  .gradientMix(linear; rgba(46, 62, 76, 0.65) 38.6%, rgba(46, 62, 76, 0.169) 59.52%, rgba(69, 95, 106, 0) 79.64%; 13.39deg);
+}
+
+.image-with-title .header a {
+  font-size: var(--image-header-font-size, @imageTitleFontSize);
```

Comment:
@tarantilis there should an uppercase transform on this title text
and I have the impression that the font size is too large.
If i look at the demo site the font weight is 500 and the font size is more
like 2.5rem instead of 2.75